packages/core/src/tools/mcp-tool.ts

```diff
@@ -53,7 +53,7 @@ export class DiscoveredMCPTool extends BaseTool<ToolParams, ToolResult> {
       DiscoveredMCPTool.allowlist.has(serverAllowListKey) ||
       DiscoveredMCPTool.allowlist.has(toolAllowListKey)
     ) {
-      return false; // server and/or tool already allow listed
+      return false; // server and/or tool already allowlisted
```

Comment:
it 2 words

Reply:
The verb allowlist is one single word: 

see https://en.wiktionary.org/wiki/allowlist

_allowlist (third-person singular simple present [allowlists](https://en.wiktionary.org/wiki/allowlists#English), present participle [allowlisting](https://en.wiktionary.org/wiki/allowlisting#English), simple past and past participle [allowlisted](https://en.wiktionary.org/wiki/allowlisted#English))_